tests/storage/restore.go

```diff
@@ -382,12 +382,16 @@ var _ = SIGDescribe("VirtualMachineRestore Tests", func() {
 				patch := []byte("[{ \"op\": \"replace\", \"path\": \"/spec/running\", \"value\": true }]")
 				vm, err := virtClient.VirtualMachine(vm.Namespace).Patch(context.Background(), vm.Name, types.JSONPatchType, patch, metav1.PatchOptions{})
 				Expect(err).ToNot(HaveOccurred())
+				Eventually(func() error {
+					_, err := virtClient.VirtualMachineInstance(vm.Namespace).Get(context.Background(), vm.Name, metav1.GetOptions{})
+					return err
+				}, 300*time.Second, 2*time.Second).Should(Succeed())
 
```

Comment:
How come you need specifically VMI existence to proceed?

Reply:
cause now the admitter checks for existence of VMI in order to reject the vmrestore

Reply:
right.. I got thrown off because the test changes are inconsistent with https://github.com/kubevirt/kubevirt/pull/13420/files#diff-a9f8dfbfa9d6d766bcefa6f3d91bae717c88c3590685409587e304511f9c09d8R397

Reply:
do you want to keep it like that or maybe this branch also has a similar solution?

Reply:
libvmops.StartVirtualMachine(vm) doesnt exist in this version there is tests.StartVirtualMachine which waits for the vm to be ready, and I only need to have the vmi exists.. so since it already does patch I just wait for a get command to succeed